### ISSUE TITLE
pipermail backend does not work

### ISSUE THREAD

**Issue by acs** [grimoirelab/arthur#9]
The pipermail perceval backend does not work with arthur.

I start arthurd and a worker:

`
(acs@dellx) ~ $ arthurd -g -d redis://localhost/8 --log-path /tmp/arthurd --no-cache && tail -f /tmp/arthurd/arthur.log
`

`
(acs@dellx) ~ $ arthurw -g -d redis://localhost/8
`

and send the task:

```
(acs@dellx) ~ $ curl -XPOST -H "Content-Type: application/json" http://localhost:8080/add -d'
{
 "tasks": [
  {
   "task_id": "http://lists.wikimedia.org/pipermail/analytics",
   "backend_args": {
    "tag": "http://lists.wikimedia.org/pipermail/analytics",
    "uri": "http://lists.wikimedia.org/pipermail/analytics"
   },
   "cache": {
    "cache": true,
    "cache_path": null,
    "fetch_from_cache": false
   },
   "backend": "pipermail",
   "scheduler": {
    "delay": 60
   }
  }
 ]
}'
Tasks added
```

and in the worker logs:

```
[2017-12-18 06:21:58,375 - rq.worker - ERROR] - arthur.errors.NotFoundError: pipermail not found
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 112, in __init__
    self._bklass = perceval.find_backends(perceval.backends)[0][backend]
KeyError: 'pipermail'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/rq/worker.py", line 700, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python3.5/dist-packages/rq/job.py", line 500, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 299, in execute_perceval_job
    rq_job.connection, qitems)
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 114, in __init__
    raise NotFoundError(element=backend)
arthur.errors.NotFoundError: pipermail not found
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 112, in __init__
    self._bklass = perceval.find_backends(perceval.backends)[0][backend]
KeyError: 'pipermail'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/rq/worker.py", line 700, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python3.5/dist-packages/rq/job.py", line 500, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 299, in execute_perceval_job
    rq_job.connection, qitems)
  File "/usr/local/lib/python3.5/dist-packages/kingarthur-0.1.1-py3.5.egg/arthur/jobs.py", line 114, in __init__
    raise NotFoundError(element=backend)
arthur.errors.NotFoundError: pipermail not found
[2017-12-18 06:21:58,375 - rq.worker - DEBUG] - Invoking exception handler <bound method Worker.move_to_failed_queue of <arthur.worker.ArthurWorker object at 0x7fbf3c447e10>>
[2017-12-18 06:21:58,375 - rq.worker - WARNING] - Moving job to 'failed' queue
```

**Comment by sduenas** [comment]
I've found the error. Perceval's function `find_backends` does not find `pipermail` as a backend.

**Comment by sduenas** [comment]
This bug was already fixed in grimoirelab/perceval#257
